### ISSUE TITLE
Does exist a way to measure voltage of EVShield + Arduino One

### ISSUE THREAD

**Issue by jabrena** [mindsensors/EVShield#2]
Hi,

I would like to know if exist a way to know what is the level of the battery connected to EVShield.

**Comment by mindsensors** [comment]
Yes, there is a library function int EVShieldBank::evshieldGetBatteryVoltage  (  ) 
 to read the voltage of EVShield battery.
The url to function documentation as follows:
http://www.openelectrons.com/EVShield/html/class_e_v_shield_bank.html#ad73253245aaae122df3a2254c4ca2069

You can read more about it here: http://www.openelectrons.com/EVShield/html/
